2 Niagara Fundamentals Exercise - Multiplexer/multiplexer-rt/src/com/tridiumuniversity/multiplexer/BMultiplex.java

```diff
@@ -0,0 +1,251 @@
+package com.tridiumuniversity.multiplexer;
+
+import javax.baja.nre.annotations.NiagaraProperty;
+import javax.baja.nre.annotations.NiagaraType;
+import javax.baja.sys.*;
+
+@NiagaraType
+@NiagaraProperty(
+        name = "in0",
+        type = "BBoolean",
+        defaultValue = "BBoolean.FALSE",
+        flags = Flags.SUMMARY
+)
+@NiagaraProperty(
+        name = "in1",
+        type = "BBoolean",
+        defaultValue = "BBoolean.FALSE",
+        flags = Flags.SUMMARY
+)
+@NiagaraProperty(
+        name = "in2",
+        type = "BBoolean",
+        defaultValue = "BBoolean.FALSE",
+        flags = Flags.SUMMARY
+)
+@NiagaraProperty(
+        name = "in3",
+        type = "BBoolean",
+        defaultValue = "BBoolean.FALSE",
+        flags = Flags.SUMMARY
+)
+@NiagaraProperty(
+        name = "s0",
+        type = "BBoolean",
+        defaultValue = "BBoolean.FALSE",
+        flags = Flags.SUMMARY
+)
+@NiagaraProperty(
+        name = "s1",
+        type = "BBoolean",
+        defaultValue = "BBoolean.FALSE",
+        flags = Flags.SUMMARY
+)
+@NiagaraProperty(
+        name = "out",
+        type = "BBoolean",
+        defaultValue = "BBoolean.FALSE",
+        flags = Flags.READONLY | Flags.SUMMARY
+)
+
+public class BMultiplex extends BComponent{
+
+  public void changed(Property p, Context cx) {
```

Comment:
I always like to use the `@Override` annotation so that the compiler complains if I'm accidentally not overriding anything.